src/test/java/htsjdk/samtools/cram/structure/ContainerTest.java

```diff
@@ -150,4 +151,103 @@ public void getSpansTest(final List<CramCompressionRecord> records,
         Assert.assertEquals(spanMap.get(expectedReferenceContext), expectedAlignmentSpan);
     }
 
+    // show that we can populate all of the slice indexing fields from the
+    // values in the container's header
+
+    // this is part of the deserialization process, and supports index creation
+
+    // single slice
+
+    @Test
+    public static void populateSlicesAndIndexingParametersTest1() {
```

Comment:
Suggest `populateSlicesAndIndexingParametersTest1` -> `populateSlicesAndIndexingParametersSingleSlice`, and `populateSlicesAndIndexingParametersTest2` -> `populateSlicesAndIndexingParametersMultiSlice`, or something similar.